### PR TITLE
#2165 - Student application - Change Separated / Divorced / Widowed Students to 'Independent' Status

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Changed conditions to accept Separated / Divorced / Widowed as "independant" from form.io;
- Changed gateway assessment workflow condition;

## Testing:
- Married student application:
image
image
image
image

- Change Separated / Divorced / Widowed Student application:
image
image
image

- Single student application:
image
image
image
image

### CODE REVIEW

File: sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn:1]: @andrepestana-aot I see similar logic in line 689 of the same file, do we need to update anything there? same for all other f.t assessment bpmn
image

**Review by ann-aot — COMMENTED** [review]
LGTM @andrepestana-aot . just a 1 comment

File: sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn
Comment (andrepestana-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn:1]: Yes, that similar logic is for "independent" sequence flow. If a student is Separated / Divorced / Widowed he/she is considered "independent". That's why it was removed from the "dependent" sequence flow condition.

Comment (andrepestana-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/fulltime-assessment-2021-2022.bpmn:1]: Done and retested. Thanks!

**Review by sh16011993 — APPROVED** [review]
LGTM @andrepestana-aot 👍 Nice work.

**Review by ann-aot — APPROVED** [review]
Thanks for doing the changes 👍 LGTM

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrepestana-aot

**Pull request merged**